### PR TITLE
ZTE-SH-CN-configure-pod-container-opaque-integer-resource-2017-09-15

### PR DESCRIPTION
**Pull request by mysunshine92**
> ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> For 1.8 Features: set Milestone to `1.8` and Base Branch to `release-1.8`
> ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>
> NOTE: Please check the “Allow edits from maintainers” box (see image below) to
> [allow reviewers to fix problems](https://help.github.com/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork/) on your patch and speed up the review process.
>
> Please delete this note before submitting the pull request.

Allow edits from maintainers checkbox

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/5469)

https://github.com/kubernetes/kubernetes.github.io/blob/master/docs/tasks/configure-pod-container/opaque-integer-resource.md

### CODE REVIEW

File: cn/docs/tasks/configure-pod-container/opaque-integer-resource.md [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+{% include feature-state-alpha.md %}
+
+{% endcapture %}
+
+
+{% capture prerequisites %}
+
+{% include task-tutorial-prereqs.md %}
+
+在做这个练习之前，请在[Advertise Opaque Integer Resources for a Node](/docs/tasks/administer-cluster/opaque-integer-resource-node/)中进行练习。
```
Comment (tengqm) [inline cn/docs/tasks/configure-pod-container/opaque-integer-resource.md]: pls translate text in links

File: cn/docs/tasks/configure-pod-container/opaque-integer-resource.md [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+它会在一个节点上配置dongle资源。
```
Comment (tengqm) [inline cn/docs/tasks/configure-pod-container/opaque-integer-resource.md]: 由于上一句翻译的不明确，这里的“它”不知所指。

File: cn/docs/tasks/configure-pod-container/opaque-integer-resource.md [hunk 3]
```diff
@@ -0,0 +1,137 @@
+---
+title: 给容器分配自定义资源
+---
+
+{% capture overview %}
+
+本页展示了如何给容器分配自定义资源。
```
Comment (tengqm) [inline cn/docs/tasks/configure-pod-container/opaque-integer-resource.md]: opaque integer resource 应直译成“非透明整型资源"

自定义资源 too generic，在其它场合可能指 extended resource （OIR的更新版本），也可能指 CRD 资源。

File: cn/docs/tasks/configure-pod-container/opaque-integer-resource.md [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+它会在一个节点上配置dongle资源。
+
+{% endcapture %}
+
+
+{% capture steps %}
+
+## 给Pod分配自定义资源
+
+为了请求一个自定义资源，需要在容器列表中包含`resources:requests`字段。
```
Comment (tengqm) [inline cn/docs/tasks/configure-pod-container/opaque-integer-resource.md]: manifest可以翻译成”清单文件“、"配置文件”，但不可以翻译成“列表”

File: cn/docs/tasks/configure-pod-container/opaque-integer-resource.md (hunk 3 above)
Comment (mysunshine92) [reply cn/docs/tasks/configure-pod-container/opaque-integer-resource.md]: ok，thank you！

File: cn/docs/tasks/configure-pod-container/opaque-integer-resource.md (hunk 1 above)
Comment (mysunshine92) [reply cn/docs/tasks/configure-pod-container/opaque-integer-resource.md]: https://github.com/kubernetes/kubernetes.github.io/blob/master/docs/tasks/configure-pod-container/opaque-integer-resource.md

Comment (tengqm) [reply cn/docs/tasks/configure-pod-container/opaque-integer-resource.md]: so you cannot understand what I was saying?

I meant the `Advertise Opaque Integer Resources for a Node` part should be translated ...

Comment (mysunshine92) [reply cn/docs/tasks/configure-pod-container/opaque-integer-resource.md]: 谢谢，我已经修改了

File: cn/docs/tasks/configure-pod-container/opaque-integer-resource.md [hunk 5]
```diff
@@ -0,0 +1,137 @@
...
+kubectl delete pod oir-demo
+```
+
+{% endcapture %}
+
+{% capture whatsnext %}
+
+### 对于应用开发者
+
+* [给容器和Pods分配内存资源](/docs/tasks/configure-pod-container/assign-memory-resource/)
```
Comment (zhangxiaoyu-zidif) [inline cn/docs/tasks/configure-pod-container/opaque-integer-resource.md]: 直接写成：
分配内存资源

File: cn/docs/tasks/configure-pod-container/opaque-integer-resource.md [hunk 6]
```diff
@@ -0,0 +1,137 @@
...
+* [给容器和Pods分配CPU资源](/docs/tasks/configure-pod-container/assign-cpu-resource/)
```
Comment (zhangxiaoyu-zidif) [inline cn/docs/tasks/configure-pod-container/opaque-integer-resource.md]: 直接写成：
分配内存资源

**Comment by mysunshine92** [comment]
@zhangxiaoyu-zidif 已经修改了

**Comment by zhangxiaoyu-zidif** [comment]
@tengqm 
for me LGTM, how about you?

File: cn/docs/tasks/configure-pod-container/opaque-integer-resource.md [hunk 7]
```diff
@@ -0,0 +1,137 @@
...
+{% include feature-state-alpha.md %}
+
+{% endcapture %}
+
+
+{% capture prerequisites %}
+
+{% include task-tutorial-prereqs.md %}
+
+在做这个练习之前，请在[在节点上配置非透明整型资源](/docs/tasks/administer-cluster/opaque-integer-resource-node/)文档中进行练习，
```
Comment (zhangxiaoyu-zidif) [inline cn/docs/tasks/configure-pod-container/opaque-integer-resource.md]: 请在[在节点上配置非透明整型资源]

连续两个 “在” 很奇怪。

**Comment by mysunshine92** [comment]
@zhangxiaoyu-zidif

**Pull request merged**